File: src/api/java/maestro/api/event/events/TabCompleteEvent.java
package maestro.api.event.events;

import maestro.api.event.events.type.Cancellable;

public final class TabCompleteEvent extends Cancellable {

    public final String prefix;
    public String[] completions;

    public TabCompleteEvent(String prefix) {
        this.prefix = prefix;
        this.completions = null;
    }
}

File: common/src/main/java/maestro/api/event/events/TabCompleteEvent.kt
package maestro.api.event.events

import maestro.api.event.events.type.Cancellable

/** Event for tab completion in chat */
class TabCompleteEvent(
    @JvmField val prefix: String,
) : Cancellable() {
    @JvmField var completions: Array<String>? = null
}
